### DISCUSSION TITLE
Pages module to show in left navigation

### DISCUSSION THREAD

**Discussion by stinhambo** [ExpressionEngine/ExpressionEngine#2948] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
It's a problem in that I have to manually add a link to the Pages module. Pages are a fundamental part of any website so not having them visible when activated seems a little odd to me.

**Describe the solution you'd like**
I'd like to see Pages have equal importance with Entries, Channels and Categories

**Describe alternatives you've considered**
As mentioned, I had to manually create a page link in the CMS.

**Comment by intoeetive** [comment]
@TomJaeger I agree with the sentiment here. I think that when installed, Pages, Structure and Pro Variables should automatically appear in the main navigation sidebar
